libr/bin/bfile.c

```diff
@@ -62,7 +62,7 @@ static void print_string(RBinFile *bf, RBinString *string, int raw) {
 				pj_o (pj);
 				pj_ks (pj, "string", string->string);
 				pj_end (pj);
-				io->cb_printf ("%s\n", pj_string (pj));
+				io->cb_printf ("%s%s\n", first ? "" : ",", pj_string (pj));
```

Comment:
using the pj api will avoid the need to add this bool first arg

Reply:
#15760